File: Another-Nikki-Web/src/types/Comment.ts
export interface comments {
    comment_id: number,
    created_time: string,
    content: string,
    user_avatar: number,
    username: string,
    root_id: number,
    parent_id: number,
    parent_name: string,
    children: comments[],
    author_avatar: string,

    // 
    replyIsVisible: boolean
    replyText: string
}

File: Another-Nikki-Server/router/model/comment.go
package model

type Comment struct {
	Content      string `json:"content"`
	AuthorID     int    `json:"author_id"`
	AuthorName   string `json:"author_name"`
	AuthorAvatar string `json:"author_avatar"`
	ParentID     int    `json:"parent_id"`
	RootID       int    `json:"root_id"`
	ParentName   string `json:"parent_name"`
}
